src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -64,7 +66,14 @@ public static void tearDown() {
         Persistence.adminUsers.removeById(adminUser.id);
```

Comment:
Some tests in this class leave AWS API keys behind. I think we need the admin user to still exist in the database in order to delete the keys, so this line should be moved to after the API user is deleted.

Reply:
You are of course correct! I have updated. Thanks.